xgi/algorithms/simpliciality.py

```diff
@@ -0,0 +1,369 @@
+from itertools import chain, combinations
+
+import numpy as np
+from scipy.special import binom
+
+from ..core import Hypergraph
+from ..utils import Trie
+
+__all__ = [
+    "edit_simpliciality",
+    "simplicial_edit_distance",
+    "face_edit_simpliciality",
+    "mean_face_edit_distance",
+    "simplicial_fraction",
+]
+
+
+def edit_simpliciality(H, min_size=2, exclude_min_size=True):
+    """Computes the edit simpliciality.
+
+    The number of edges needed to be added
+    to a hypergraph to make it a simplicial complex.
+
+    Parameters
+    ----------
+    H : xgi.Hypergraph
+        The hypergraph of interest
+    min_size: int, default: 2
+        The minimum hyperedge size to include when
+        calculating whether a hyperedge is a simplex
+        by counting subfaces.
+    exclude_min_size : bool, optional
+        Whether to include minimal simplices when counting simplices, by default True
+
+    Returns
+    -------
+    float
+        The edit simpliciality
+
+    See Also
+    --------
+    simplicial_edit_distance
+
+    References
+    ----------
+    "The simpliciality of higher-order order networks"
+    by Nicholas Landry, Jean-Gabriel Young, and Nicole Eikmeier,
+    *EPJ Data Science* **13**, 17 (2024).
+    """
+    return 1 - simplicial_edit_distance(
+        H, min_size=min_size, exclude_min_size=exclude_min_size
+    )
+
+
+def simplicial_edit_distance(H, min_size=2, exclude_min_size=True, normalize=True):
+    """Computes the simplicial edit distance.
+
+    The number of edges needed to be added
+    to a hypergraph to make it a simplicial complex.
+
+    Parameters
+    ----------
+    H : xgi.Hypergraph
+        The hypergraph of interest
+    min_size: int, default: 2
+        The minimum hyperedge size to include when
+        calculating whether a hyperedge is a simplex
+        by counting subfaces.
+    exclude_min_size : bool, optional
+        Whether to include minimal simplices when counting simplices, by default True
```

Comment:
The name is `exclude_min_size`, but the documentation reads "Whether to _include_...default `True`", which makes my head spin a bit! Maybe reword for clarity?

Reply:
Good call. I will correct everywhere.